test/integration_test/specs/mysql-1-pvc/mysql.yaml

```diff
@@ -1,3 +1,30 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: volume-secrets
+type: Opaque
+data:
+  mysql-secret: WW91IHNuZWFreSBsaXR0bGUgcGlnbGV0IQ==
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+   name: mysql-data-enc
+   labels:
+     app: mysql
+   annotations:
+     volume.beta.kubernetes.io/storage-class: px-mysql-sc
+     px/secret-name: volume-secrets
+     px/secret-namespace: mysql-1-pvc-mysql
```

Comment:
The secret won't be in that namespace, it is auto-generated for most tests

Reply:
Fixed.